tests/app/v2/notifications/test_get_notifications.py

```diff
@@ -422,7 +422,8 @@ def test_get_all_notifications_filter_by_status_invalid_status(client, sample_no
     assert json_response['status_code'] == 400
```

Comment:
Should we add a new test which sets the status to pending-virus-check and virus-scan-failed (or add them to a parametized test) to check they don't do things that are not expected

i.e. A virus-scan-failed might appears on get message status but pending-virus-check wouldn't get sent etc?